MozillaFoundation/foundation.mozilla.org
# Create taxonomy classes

# Description

Create taxonomy classes for RCC Playbook.

Link to sample test page:
Related PRs/issues: #10425 

# Checklist



**Tests**

**Changes in Models:**

**Documentation:**

**Testing**

1. Log into the CMS
2. Go to Snippets
3. Add/remove/edit `RCC Content Types`, `RCC Curricular Areas` and `RCC Topics`
4. Make sure to check ACs in #10425
